### PR TITLE
Fix: text margin and alignment

### PR DESCRIPTION
**Pull request by abhishekmmgn**
**Notes for Reviewers**

This PR fixes #
image

Now:
image

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 5d96b8a3915a4b5fa2684ca98af9be413bd7d664 at: https://6627ff68235ba731cfeffe2c--meshery-docs-preview.netlify.app

**Comment by codeKraken19** [comment]
@abhishekmmgn  is the scroll offset issue fixed here ?

File: docs/pages/reference/error-codes.md [hunk 1]
```diff
@@ -45,7 +46,7 @@ td {
 }
 
 .tbl-body-row .error-name-code{
-  display:flex;
+  /* display:flex; */
```
Comment (leecalcote) [inline docs/pages/reference/error-codes.md:49]: why?

File: docs/_sass/table.scss [hunk 2]
```diff
@@ -20,7 +20,7 @@ table th {
 .td-box .row.section > table td,
 .td-content > table td {
   padding: 1rem;
-  vertical-align: top;
```
Comment (leecalcote) [inline docs/_sass/table.scss:23]: Did you confirm that these changes don't negatively affect other table uses, like the integration test results?

**Comment by abhishekmmgn** [comment]
> @abhishekmmgn is the scroll offset issue fixed here?

No, what's that?

**Comment by l5io** [comment]
🚀 Preview for commit be7480ff6e86735d43a56534b883dbe70b39e1c3 at: https://6635af6b783c73948ff7291f--meshery-docs-preview.netlify.app

**Comment by iArchitSharma** [comment]
These changes were already incorporated in @abhishekmmgn's other [PR](https://github.com/meshery/meshery/pull/10765), so I'm closing this PR

**Pull request closed without merging**